static/src/stylesheets/amp/_article-tones.scss

```diff
@@ -157,7 +166,8 @@
     color: #21a2bc;
 }
 
-.tonal--tone-analysis .tone-colour {
+.tonal--tone-analysis .tone-colour,
+.tonal__head--tone-special-report .content__section-label__link {
```

Comment:
Instead of adding the colour to `.content__section-label__link`, could we keep it on `.tone-colour`? Try maybe:

```
.tonal--tone-analysis, .tonal__head--tone-special-report {
   .tone-colour {
...
```